__testfixtures__/fourteen-testing-api/setup-route-empty-needs-test.input.js

```diff
@@ -0,0 +1,22 @@
+import { describe, it, beforeEach } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import hbs from 'htmlbars-inline-precompile';
+
+describe('LoadingIndexRoute', function() {
+  setupTest('route:loading/index', {
+    // needs: ['service:ajax']
```

Comment:
Does this comment serve a purpose to the test?  Like, is this to ensure comments are also removed?

Reply:
@rondale-sc indeed. I wanted to have a sample of what you might find in an older `ember-mocha` codebase. Not strictly necessarily, but fulfilling a goal of mine to test against real world snippets.